### ISSUE TITLE
Option to clear quickfix list before filling it

### ISSUE THREAD

**Issue by ysangkok** [inkarkat/vim-IndentConsistencyCopAutoCmds#1]
I use this plugin with the WriteCheck, but if I save a file with error while having an old resolved quickfix, it seems like the old fix is not overwritten. I found that I need to run `:cexpr []` before saving, but this defeats the purpose of not thinking about indent before the cop tells me about it. So it would be nice if this autocmd library also had an option to clear the quickfix list first.

Thanks a lot!

**Comment by inkarkat** [comment]
Right, [vim-IndentConsistencyCop](/inkarkat/vim-IndentConsistencyCop) currently only populates the quickfix / location list, but does not clean up if the result becomes positive. The problem is that another plugin or the user may have overwritten the list, and we don't want to clear these unrelated entries. Recent Vim versions can associate a _context_ with a quickfix list; that would allow the plugin to recognize its own list, and clear it only then.
